applications/nrf_desktop/src/modules/Kconfig.hotfixes

```diff
@@ -8,12 +8,22 @@ menu "Hotfix set"
 
 config DESKTOP_HFCLK_LOCK_ENABLE
 	bool "Keep HF clock enabled"
+	depends on !SOC_SERIES_NRF54HX
 	help
-	  Keeping the HF clock enabled is needed to reduce the latency before
-	  the first packet (in a row) is transmitted over the BLE.
-	  If disabled a startup delay of around 1.5 ms will be added to
-	  overall latency of the first packet.
-	  If enabled current consumption is increased.
+	  The HF clock needs to be enabled during data transmission over BLE.
+	  Enabling the HF clock right before the data transmission adds an extra
+	  latency of around 1.4 ms (0.85 ms in case of nRF54L Series SoCs) for
+	  the first packet (in a row) that is transmitted. The extra latency is
+	  caused by the clock startup latency. Keeping the HF clock enabled all
+	  the time removes the extra latency caused by the clock startup at the
+	  cost of increased power consumption.
+
+	  The HF clock lock hotfix is not supported for nRF54H Series SoCs. For
+	  these SoCs, part of the Bluetooth stack runs on radio core and an
+	  application runs on application core. Keeping the HF clock enabled by
+	  the application cannot be used to mitigate extra latency caused by
+	  the HF clock startup, because an application cannot notify the
+	  Bluetooth stack that HF clock is already enabled.
```

Comment:
```suggestion
	  The HF clock lock hotfix is not supported for nRF54H Series SoCs. For
	  these SoCs, part of the Bluetooth stack runs on the radio core and an
	  application runs on the application core. Keeping the HF clock enabled by
	  the application cannot be used to mitigate extra latency caused by
	  the HF clock startup, because an application cannot notify the
	  Bluetooth stack that HF clock is already enabled.
```

Reply:
Rephrased